doc-content/enterprise-only/springboot/spring-boot-persistence-proc.adoc

```diff
@@ -0,0 +1,38 @@
+[id='spring-boot-persistence-proc_{context}']
+= Configuring plugable variable persistence
+
+You can provide an arbitrary entity manager for configured process variable persistence in your {PRODUCT} Spring Boot application. To do this, add named beans during the object marshalling strategy resolution. This configuration defines a second entity manager factory based on a qualifier second data source. This process will not interfere with the primary data source.
```

Comment:
- Shouldn't be worth mentioning the configuration as a primary and second data sources by using the `application.properties` file in spring-boot? as this is about multiple data sources

- What about using and showing the persistence and orm mapping files - I can't see the `org.drools.persistence.jpa.marshaller.MappedVariable` use.

@elguardian - what do you think

Reply:
@afalhambra Enrique says he doesnt' have time this week to answer questions. He says that you can answer all of the questions.